bucket/filezilla.json

```diff
@@ -15,10 +15,15 @@
     },
     "extract_dir": "FileZilla-3.43.0",
     "pre_install": [
-        "$file = 'fzdefaults.xml'",
-        "if (-not (Test-Path \"$persist_dir\\$file\")) {",
-        "    Copy-Item \"$dir\\docs\\fzdefaults.xml.example\" \"$dir\\fzdefaults.xml\"",
-        "}"
+        "@\"",
+        "<?xml version=\"1.0\" encoding=\"UTF-8\" standalone=\"yes\" ?>",
+        "<FileZilla3>",
+        "  <Settings>",
+        "    <Setting name=\"Config Location\">$persist_dir</Setting>",
+        "    <Setting name=\"Disable update check\">1</Setting>",
+        "  </Settings>",
+        "</FileZilla3>",
+        "\"@ | Out-File -Encoding utf8 \"$dir\\fzdefaults.xml\""
```

Comment:
UTF8 is UTF8-with-bom in powershell 5

Reply:
thanks i was not aware of that. Filezilla support UTF8 with BOM so it was not a problem. Anyway I changed to ascii (no UTF-8 with no BOM in PowerShell 5) since now Config location is assured to be ascii only, and so the entire fzdefault.xml file